### PR TITLE
Document use of GitHub assignee

### PR DESCRIPTION
**Pull request by prprabhu-ms**
Show how to assign reviews by assigning reviewers for this PR.

### CODE REVIEW

**Comment by prprabhu-ms** [comment]
@alkwa-msft @carocao-msft PTAL.

**Review by alkwa-msft — APPROVED** [review]

File: docs/references/quick-code-reviews.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+We use the assignee role to track which reviewers are responsible for (speedy) code reviews.
+
+As an author, you should
+
+- Add two [assignees](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users) for your PRs.
+- Assign new folks if the number of assignees falls below 2 and you still need two approvals.
+- Feel empowered to ping assignees if you do not have reviews in a reasonable amount of time.
+  It is their responsibility to give you timely reviews (see point below about assignee load-shedding).
+
+As an asignee, you should:
```
Comment (alkwa-msft) [inline docs/references/quick-code-reviews.md:28]: assignee

**Comment by alkwa-msft** [comment]
should we talk about how many assignees are required for cherry-pick PRs? I thought something was wrong but it turned out for cherry-picking for a release we were fine with just one. Perhaps the documentation can confirm this fact?

@prprabhu-ms

**Review by dmceachernmsft — APPROVED** [review]

**Comment by prprabhu-ms** [comment]
> should we talk about how many assignees are required for cherry-pick PRs? I thought something was wrong but it turned out for cherry-picking for a release we were fine with just one. Perhaps the documentation can confirm this fact?
> 
> @prprabhu-ms

I'll keep the discussion of release branches out of this doc for now.

We haven't protected the release branches enough so that it is possible to merge without 2 *approvals*.

Assignees are orthogonal to protecting branches via required approvals.

**Pull request merged**